Makefile

```
@@ -18,8 +18,11 @@ dryRelease:
 doRelease:
 	./gradlew bintrayUpload -PbintrayUser="$(BINTRAY_USERNAME)" -PbintrayKey="$(BINTRAY_KEY)" -PmavenCentralUser="$(MAVEN_USER)" -PmavenCentralPassword="$(MAVEN_PASS)" -PmavenCentralSync=true -PdryRun=false --info
 
+distZip:
+	./gradlew distZip
+
 # deep clean, build and deploy to bintray, jcenter and maven central
-release: clean compile dryRelease doRelease
+release: clean compile dryRelease doRelease distZip
```

```suggestion
release: clean compile dryRelease distZip
```

I think we should drop `doRelease` here then

---

@marandaneto I thought of keeping it because it's a dryRun but it tests the deployment process.
Since the distZip just packages it, we'll only find out if we broke release once the work craft workflow is done.

---

@bruno-garcia wrong, `dryRelease` is the dryRun, `doRelease` will actually call the APIs
this make command actually does the `dryRelease` to see if everything is alright and only then `doRelease`

---

got it. Ill fix on aother PR

---

https://github.com/getsentry/sentry-java/commit/17eb85672c661630d97ed57462d62488883a0f07